source/edu/oakand/test/middleware01/PointTest.java

```diff
@@ -0,0 +1,28 @@
+package edu.oakland.test.middleware01;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import edu.oakland.helper.middleware01.Point;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("JUnit 5 Point Test")
+public class PointTest {
+
+  private int coordX = (int) (Math.random() * 100);
```

Comment:
Maybe this should not be random, don't want it the same as y.

Reply:
Random works well enough.